### PR TITLE
docs(PPDSC-2704): update stories

### PR DESCRIPTION
**Pull request by mstuartf**
[PPDSC-2704](https://nidigitalsolutions.jira.com/browse/PPDSC-2704)

**What**

1. Background - why this is needed
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

[PPDSC-2704]: https://nidigitalsolutions.jira.com/browse/PPDSC-2704?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2704-form-input-stories
  Storybook: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2704-form-input-stories/storybook

File: src/form/__tests__/form-input.stories.tsx [hunk 1]
```diff
@@ -512,312 +525,211 @@ export const StoryFormFieldset = () => (
...
-            <FormInput
-              name="small-checkbox"
-              rules={{
-                required: 'Required field',
-              }}
-            >
               <FormInputCheckbox
-                label="I want to hear about more offers"
-                value="tc"
+                id="interests-Science"
```
Comment (mutebg) [inline src/form/__tests__/form-input.stories.tsx:706]: Do you need to pass the ID everywhere, or we have a bug?

**Review by mutebg — APPROVED** [review]

File: src/form/__tests__/form-input.stories.tsx (hunk 1 above)
Comment (mstuartf) [reply src/form/__tests__/form-input.stories.tsx:706]: Yeah, I was getting a11y errors without. I think the issue was the ids for the two sets of radios / checkboxes were the same unless I set them manually.

Comment (mutebg) [reply src/form/__tests__/form-input.stories.tsx:706]: I see, I know why is happening, might create a ticket for that.

Comment (JohnTParsons) [reply src/form/__tests__/form-input.stories.tsx:706]: IDs should be lower-case I think, eg id="interests-science"

Comment (mstuartf) [reply src/form/__tests__/form-input.stories.tsx]: updated

**Review by mutebg — APPROVED** [review]

**Review by JohnTParsons — APPROVED** [review]

**Pull request merged**